### ISSUE TITLE
Sometimes getting exception while starting dashboard.sh in ubuntu

### ISSUE THREAD

**Issue by prashantkt10** [wso2/product-sp#892]
**On starting dashboard.sh executable in ubuntu, I'm getting this exception sometimes. What could be possible reason for this exception ??**
```
[2018-12-04 13:49:46,071]  INFO {org.wso2.transport.http.netty.listener.ServerConnectorBootstrap$HTTPServerConnector} - HTTP(S) Interface starting on host 0.0.0.0 and port 9643
[2018-12-04 13:50:09,497]  WARN {org.wso2.msf4j.internal.MSF4JHttpConnectorListener} - Unmapped exception java.lang.NullPointerException
	at org.wso2.carbon.analytics.auth.rest.api.impl.LoginApiServiceImpl.loginAppNamePost(LoginApiServiceImpl.java:110)
	at org.wso2.carbon.analytics.auth.rest.api.LoginApi.loginAppNamePost(LoginApi.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.wso2.msf4j.internal.router.HttpMethodInfo.invokeResource(HttpMethodInfo.java:187)
	at org.wso2.msf4j.internal.router.HttpMethodInfo.invoke(HttpMethodInfo.java:143)
	at org.wso2.msf4j.internal.MSF4JHttpConnectorListener.dispatchMethod(MSF4JHttpConnectorListener.java:218)
	at org.wso2.msf4j.internal.MSF4JHttpConnectorListener.lambda$onMessage$57(MSF4JHttpConnectorListener.java:129)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by Niveathika92** [comment]
@prashantkt10 Did the Stream Processor start correctly? Could you paste the full log to verify this?

**Comment by Niveathika92** [comment]
@prashantkt10 We have identified this issue and fixed with the above PR. This fix will be available in next week milestone release.

**Comment by prashantkt10** [comment]
Thank you @Niveathika92 !

**Comment by Niveathika92** [comment]
Closing as this fix is available in 4.4.0-m12 release
